samcli/lib/utils/resources.py

```diff
@@ -59,7 +59,7 @@
     AWS_SERVERLESS_STATEMACHINE: ["DefinitionUri"],
     AWS_APPSYNC_GRAPHQLSCHEMA: ["DefinitionS3Location"],
     AWS_APPSYNC_RESOLVER: ["RequestMappingTemplateS3Location", "ResponseMappingTemplateS3Location"],
```

Comment:
@roh-hitiq Can you add `CodeS3Location` to this list as well?

Reply:
This has been added